pyshield/_config.py

```diff
@@ -22,16 +22,31 @@ class PHYSICS_PACKAGES(Enum, metaclass=MetaEnumStr):
 @dataclasses.dataclass
 class SurfaceConfig:
     do_z0_hwrf15: bool = DEFAULT_BOOL
+    """flag to use z0 scheme from 2015 HWRF"""
```

Comment:
Am hoping to eventually replace all these z0 scheme bools with an integer or enum switch

Reply:
That's definitely something we can do!

Reply:
There's a feature coming when we got to python 3.11 that will allow Enum in stencils